javaslang-benchmark/src/test/java/javaslang/benchmark/BenchmarkResultAggregator.java

```diff
@@ -2,99 +2,94 @@
 
 import javaslang.*;
 import javaslang.collection.*;
+import javaslang.collection.List;
+import javaslang.collection.Map;
+import org.intellij.lang.annotations.RegExp;
 import org.openjdk.jmh.results.*;
 
 import java.text.DecimalFormat;
-import java.util.Collection;
+import java.util.*;
 
 public class BenchmarkResultAggregator {
     private static final DecimalFormat DECIMAL_FORMAT = new DecimalFormat("#0.00");
 
-    public static void displayRatios(Collection<RunResult> runResultsCollection) {
-        final Array<RunResult> runResults = Array.ofAll(runResultsCollection);
-
-        if (runResults.size() > 1) {
-            printHeader(runResults);
-            for (Tuple2<String, Array<RunResult>> group : getGroupsBasedOnBenchmarkPrefix(runResults)) {
-                final Multimap<Integer, Tuple2<String, Double>> results = aggregate(group._2);
-                final Multimap<String, Double> ratios = getRatios(results);
-                if (!ratios.isEmpty()) {
-                    System.out.println("Group '" + group._1 + "':");
-
-                    final Seq<String> output = getFormattedRatios(ratios);
-                    output.forEach(System.out::println);
-                }
+    public static void displayRatios(Collection<RunResult> runResultsCollection, @RegExp String resultFilter) {
```

Comment:
Changed the result aggregator to be similar to `JMH`s style, used by @eduardmanas also :)